build.cake

```diff
@@ -747,6 +767,15 @@ Task("Publish-NuGet")
     Information("Publish-NuGet Task failed, but continuing with next Task...");
 });
 
+Task("Publish-Coverage")
+    .IsDependentOn("Run-Unit-Tests")
+    .WithCriteria(() => isRunningOnAppVeyor)
```

Comment:
We want PR coverage, at least if we do CodeCov. CodeCov provides a GitHub PR check for how coverage changes. I don't know what Coveralls provides in that department.